AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 balcaorural.com.br##.j-cookieAlert
+dlford.io###gatsby-focus-wrapper > div > footer
```

Comment:
It's better to use ext-css here

Reply:
No need. I tried no FP.

Reply:
@Sergey-Lyapin Fixed.

Reply:
In both cases footer is broken
![image](https://user-images.githubusercontent.com/8361299/110141501-e2b9bd80-7ddd-11eb-9648-602390816fcb.png)
JS rule can be used here to work with LocalStorage(you can find similar rules in Annoyances filter)
![image](https://user-images.githubusercontent.com/8361299/110141705-18f73d00-7dde-11eb-91c3-e9b44db80052.png)

Reply:
`dlford.io#%#(function() { try { if ('undefined' != typeof localStorage) { localStorage.setItem('closedPrivacyNotice', 'true') } } catch (ex) {} })();`

Reply:
@Sergey-Lyapin Where did you find this? `() { try { if ('undefined' != typeof localStorage) {`? Isn't this enough? `localStorage.setItem('closedPrivacyNotice', 'true')`? That's a pure javascript.